chapters/logging.rst

```diff
@@ -0,0 +1,242 @@
+==============================
+Logging configuration in Jina
+==============================
+
+.. meta::
+   :description: A guide on logging configuration in Jina
+   :keywords: Jina, logging
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Motivation
+-------------------
+
+In order to better understand, monitor, and debug the running and lifetime of Jina's peas, pods, and flow, Jina offers logging to help log the messages.
+
+Before you start
+-------------------
+You have installed the latest stable release of Jina according to the instructions found `here <https://docs.jina.ai/chapters/core/setup/index.html>`_ .
+
+
+Overview
+-------------------
+
+Jina logging message at first glance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Let's start from an logging example. Jina logs messages with different formats in different situations. The example below follows the most commonly used format:
+
+.. highlight:: bash
+.. code-block:: bash
+
+   pod1@25908[I]:starting jina.peapods.runtimes.zmq.zed.ZEDRuntime...
+   pod1@25908[I]:input tcp://0.0.0.0:63915 (PULL_BIND) output tcp://0.0.0.0:63916 (PUSH_BIND) control over tcp://0.0.0.0:63914 (PAIR_BIND)
+   pod1@25903[S]:ready and listening
+
+It follows the following format:
```

Comment:
Replace with: 
This is how the message is composed :

Reply:
done